queue/driver_mongo.go

```diff
@@ -680,6 +680,12 @@ func (d *mongoDriver) Put(ctx context.Context, j amboy.Job) error {
 	return d.PutMany(ctx, []amboy.Job{j})
 }
 
+// PutMany enqueues multiple jobs on the queue.
+// Returns an error if any of the jobs cannot be inserted. If a non-duplicate-job error
+// is encountered it is returned. If every job is a duplicate job error then if any
```

Comment:
nit: "If every error is a duplicate job error"

Reply:
done